pkg/codegen/internal/test/testdata/output-funcs/dotnet/funcWithConstInput.cs

```diff
@@ -0,0 +1,31 @@
+// *** WARNING: this file was generated by . ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.MadeupPackage.Codegentest
+{
+    public static class FuncWithConstInput
+    {
+        /// <summary>
+        /// Codegen demo with const inputs
+        /// </summary>
+        public static Task InvokeAsync(FuncWithConstInputArgs? args = null, InvokeOptions? options = null)
+            => Pulumi.Deployment.Instance.InvokeAsync("madeup-package:codegentest:funcWithConstInput", args ?? new FuncWithConstInputArgs(), options.WithVersion());
+    }
```

Comment:
Why no Invoke variant?

Reply:
So. This function does not have a return type, it's returning void. When I was developing this for the other languages, I was slightly intimidated by covering these cases correctly, so I did a search on AWS and Azure, and having found *no functions* that return void, I decided to just not handle it.

Now that I have automated testing on the gen'd code, I'm less intimidated about it, and I can bring this feature in as a subsequent set of PRs across languages. However d'you know actual use case that we really need it? Did I miss some Azure or AWS functions with void returns, or have some been added lately?

Reply:
No, I don't really know why such a function would be needed, as functions are supposed to be side-effect-free. I'm fine with ignoring this use case.

Reply:
I missed that - we need to upgrade our specs or else I need to RTFM. In that case this is just a bad test case. I can later on update the test case to just test the paths with output values we care about.